src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -160,8 +160,8 @@ def test_WINNF_FT_S_WDB_1(self, config_filename):
                                 config['palDatabaseConfig']['filePath'])
 
     # Inject the PAL database URL into the SAS UUT.
-    self._sas_admin.InjectDatabaseUrl(pal_database.getBaseUrl()+
-                                      config['palDatabaseConfig']['fileUrl'])
+    self._sas_admin.InjectDatabaseUrl({'type': 'PAL', 'url': pal_database.getBaseUrl()+
+                                       config['palDatabaseConfig']['fileUrl']})
```

Comment:
Looks like we need to append the file name to this URL as well (thanks Javi!).

Reply:
The fileUrl already included the file name in it.

Reply:
I was wrong, now fixed.